deleter.py

```diff
@@ -0,0 +1,62 @@
+# import statements should be here
+import shutil
+# import codecs
+# import json
+
+
+class Node:
+    def __init__(self, guid):
+        self.folder_path = self.get_path_from_guid(guid)
+        self.pathFound = True
+
+    def get_path_from_guid(self, guid):
+        path = 'archive/' + guid
+        return path
+
+
+class Deletion:
```

Comment:
Maybe "Deleter" is a better name for the class. It's more consistent with the filename.